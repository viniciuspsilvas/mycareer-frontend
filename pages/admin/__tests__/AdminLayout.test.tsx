import { render, screen } from '@testing-library/react'
import AwardsPage from '../index'

describe('AdminLayout', () => {
  test('renders unchanged', () => {
    const { container } = render(<AwardsPage />)
    expect(container).toMatchSnapshot()
  })

  test('should render correctly', () => {
    render(<AwardsPage />)

    const link = screen.getByRole('link', { name: /My Career/i })
    expect(link).toBeInTheDocument()
  })
})
