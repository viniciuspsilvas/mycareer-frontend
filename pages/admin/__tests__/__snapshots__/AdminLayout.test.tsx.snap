// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminLayout renders unchanged 1`] = `
<div>
  <div
    class="md:flex flex-col md:flex-row md:min-h-screen w-full"
  >
    <div
      class="flex flex-col w-full md:w-64 text-gray-700 bg-gray-400 dark-mode:text-gray-200 dark-mode:bg-gray-800 flex-shrink-0"
    >
      <div
        class="flex-shrink-0 px-8 py-4 flex flex-row items-center justify-between"
      >
        <a
          class="text-lg font-semibold tracking-widest text-gray-900 uppercase rounded-lg dark-mode:text-white focus:outline-none focus:shadow-outline"
          href="#"
        >
          My Career
        </a>
        <button
          class="rounded-lg md:hidden focus:outline-none focus:shadow-outline"
        >
          <svg
            class="w-6 h-6"
            fill="currentColor"
            viewBox="0 0 20 20"
          >
            <path
              clip-rule="evenodd"
              d="M3 5a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM3 10a1 1 0 011-1h12a1 1 0 110 2H4a1 1 0 01-1-1zM9 15a1 1 0 011-1h6a1 1 0 110 2h-6a1 1 0 01-1-1z"
              fill-rule="evenodd"
              x-show="!open"
            />
            <path
              clip-rule="evenodd"
              d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
              fill-rule="evenodd"
              x-show="open"
            />
          </svg>
        </button>
      </div>
      <nav
        class="flex-grow md:block px-4 pb-4 md:pb-0 md:overflow-y-auto"
      >
        <a
          class="block px-4 py-2 mt-2 text-sm font-semibold text-gray-900 bg-transparent rounded-lg dark-mode:bg-transparent dark-mode:hover:bg-gray-600 dark-mode:focus:bg-gray-600 dark-mode:focus:text-white dark-mode:hover:text-white dark-mode:text-gray-200 hover:text-gray-900 focus:text-gray-900 hover:bg-gray-200 focus:bg-gray-200 focus:outline-none focus:shadow-outline undefined
       false"
          href="/admin/awards"
          id="Awards_nav_link"
        >
          Awards
        </a>
      </nav>
    </div>
    <main
      class="flex flex-col w-full bg-white dark:bg-gray-600 "
    >
      <div
        class="flex w-full mx-auto md:p-8 "
      >
        <div
          class="flex flex-col w-full h-full"
        >
          <h2>
            Super-secret admin stuff for Super Users.
          </h2>
        </div>
      </div>
    </main>
  </div>
</div>
`;
